src/main/scala/scalismo/image/filter/DiscreteImageFilter.scala

```diff
@@ -30,7 +31,7 @@ object DiscreteImageFilter {
    * Computes a (signed) distance transform of the image.
    * @note The value that is returned is not the euclidean distance unless the image has unit spacing. Even worse, the distance might depend on the spacing of the image.
    */
-  def distanceTransform[D: NDSpace: CanConvertToVtk: DiscreteScalarImage.Create, A: Scalar: ClassTag: TypeTag](img: DiscreteScalarImage[D, A]): DiscreteScalarImage[D, Float] = {
+  def distanceTransform[D: NDSpace: CanConvertToVtk: BSplineImageInterpolator.Create, A: Scalar: ClassTag: TypeTag](img: DiscreteScalarImage[D, A]): DiscreteScalarImage[D, Float] = {
```

Comment:
This context bound is just there because internally a b-spline image interpolator is used. As the function computes a distance image, linear interpolation (i.e. b-spline of order 1) is always a good choice.